### PR TITLE
 warn and warn_ban counts for channel log

### PR DESCRIPTION
**Pull request by 1maverick1**
Channel log with warn and warn_ban counts

### CODE REVIEW

File: tg_bot/modules/warns.py [hunk 1]
```diff
@@ -56,9 +56,10 @@ def warn(user: User, chat: Chat, reason: str, message: Message, warner: User = N
                      "\n#WARN_BAN" \
                      "\n<b>Admin:</b> {}" \
                      "\n<b>User:</b> {}" \
-                     "\n<b>Reason:</b> {}".format(html.escape(chat.title),
+                     "\n<b>Reason:</b> {}"\
+                     "\n<b>Counts:</b> <code>{}/{}</code>".format(html.escape(chat.title),
                                                   warner_tag,
-                                                  mention_html(user.id, user.first_name), reason)
+                                                  mention_html(user.id, user.first_name), reason, num_warns, limit)
```
Comment (PaulSonOfLars) [inline tg_bot/modules/warns.py]: Make sure that all format args are correctly aligned to PEP8 standards. (and if this takes you over 120characters, get a new line

File: tg_bot/modules/warns.py [hunk 2]
```diff
@@ -73,9 +74,10 @@ def warn(user: User, chat: Chat, reason: str, message: Message, warner: User = N
                      "\n#WARN" \
                      "\n<b>Admin:</b> {}" \
                      "\n<b>User:</b> {}" \
-                     "\n<b>Reason:</b> {}".format(html.escape(chat.title),
+                     "\n<b>Reason:</b> {}"\
+                     "\n<b>Counts:</b> <code>{}/{}</code>".format(html.escape(chat.title),
                                                   warner_tag,
-                                                  mention_html(user.id, user.first_name), reason)
+                                                  mention_html(user.id, user.first_name), reason, num_warns, limit)
```
Comment (PaulSonOfLars) [inline tg_bot/modules/warns.py]: same

**Comment by Shabier** [comment]
PEP8 state you should limit all lines to a maximum of 79 characters. Consider using a new line. 
Reference: https://www.python.org/dev/peps/pep-0008/#maximum-line-length

**Comment by PaulSonOfLars** [comment]
80 characters is ridiculous in an indent based language, sticking to 120 is more than sensible. We aren't using CRT monitors from the 70s, so I think we can safely assume we have enough screen real-estate to use more than 80.
Either way, I've been sticking to 120 across the entire project so its fine.

**Comment by 1maverick1** [comment]
I've added a new line to comply with standards as requested.

File: tg_bot/modules/warns.py [hunk 3]
```diff
@@ -73,9 +75,11 @@ def warn(user: User, chat: Chat, reason: str, message: Message, warner: User = N
                      "\n#WARN" \
                      "\n<b>Admin:</b> {}" \
                      "\n<b>User:</b> {}" \
-                     "\n<b>Reason:</b> {}".format(html.escape(chat.title),
+                     "\n<b>Reason:</b> {}"\
+                     "\n<b>Counts:</b> <code>{}/{}</code>".format(html.escape(chat.title),
                                                   warner_tag,
-                                                  mention_html(user.id, user.first_name), reason)
+                                                  mention_html(user.id, user.first_name), 
+                                                  reason, num_warns, limit)
```
Comment (PaulSonOfLars) [inline tg_bot/modules/warns.py]: your arguments still aren't aligned to the format call.

File: tg_bot/modules/warns.py [hunk 4]
```diff
@@ -79,7 +79,7 @@ def warn(user: User, chat: Chat, reason: str, message: Message, warner: User = N
                      "\n<b>Counts:</b> <code>{}/{}</code>".format(html.escape(chat.title),
                                                   warner_tag,
                                                   mention_html(user.id, user.first_name), 
-                                                  reason, num_warns, limit)
+                                                  			reason, num_warns, limit)
```
Comment (PaulSonOfLars) [inline tg_bot/modules/warns.py]: dont mix tabs and spaces, stick to spaces

File: tg_bot/modules/warns.py [hunk 5]
```diff
@@ -79,7 +79,7 @@ def warn(user: User, chat: Chat, reason: str, message: Message, warner: User = N
                      "\n<b>Counts:</b> <code>{}/{}</code>".format(html.escape(chat.title),
                                                   warner_tag,
                                                   mention_html(user.id, user.first_name),
```
Comment (PaulSonOfLars) [inline tg_bot/modules/warns.py]: if you move the call to format, it's your job to align the OTHER arguments too. Not just the ones you've added.

File: tg_bot/modules/warns.py (hunk 4 above)
Comment (PaulSonOfLars) [reply tg_bot/modules/warns.py]: Additionally, this still isn't aligned to the opening bracket, probably due to tabs.

File: tg_bot/modules/warns.py (hunk 5 above)
Comment (PaulSonOfLars) [reply tg_bot/modules/warns.py]: consider the consistency.

**Comment by 1maverick1** [comment]
Sorry for the delay, just aligned other arguments as well. This time it's used only with spaces to align.

**Pull request merged**